### ISSUE TITLE
Update for BE Dims?

### ISSUE THREAD

**Issue by Desbrina** [cfogrady/DIM-Modifier#11]
Hi,

Is there any intention to update the application to support BE dims?

**Comment by cfogrady** [comment]
Short answer: Yes.
Long answer:
There is new security in the BE DIMs that I haven't had a chance to look at yet. I don't know if they prevent modification of certain areas or the entire DIM. The functionality is also different enough in places that some views on my application may need changes or need to be redone entirely.

If I'm able to make the update as I hope, don't expect it for a while (probably a month or two). I don't actually have the device of BE DIMs myself yet (still waiting on shipping). And similarly to edits made for the VB DIMs, don't expect those BE DIMs to work with the App.

**Comment by Desbrina** [comment]
Thanks, I was more looking at trying to get the evolution details as the app doesn't list some of them

**Comment by cfogrady** [comment]
I am pretty sure humulos is working on this. He'll probably have it before I do since that is what his website primarily focuses on.

I don't think there is a direct link yet, but you should be able to find the work in progress for that here: https://humulos.com/digimon/vbbe/

**Comment by LinkAlmeida** [comment]
I made a backup of my Imperialdramon BE card. I can send it to you for testing if needed.

**Comment by Desbrina** [comment]
I have backups of all 3 dims so far if wanted

**Comment by cfogrady** [comment]
I appreciate the offer. I already have the images for all three digimon cards :)
I've already started work on this, and I've already got most of the new card specification documented. Juggling between this, and defeating the sprite signing so I can have english names on newer cards and, ideally, custom BEM cards.

**Comment by LinkAlmeida** [comment]
> I appreciate the offer. I already have the images for all three digimon cards :)
> I've already started work on this, and I've already got most of the new card specification documented. Juggling between this, and defeating the sprite signing so I can have english names on newer cards and, ideally, custom BEM cards.

Cool, i've been doing some translation work on the kamen rider and ultraman dims since no one seems to be doing them. Also can't wait to make some express BEMs if there is no weird protection on the cards

**Comment by cfogrady** [comment]
Done as part of 2.0
